src/it/scala/com/wavesplatform/it/transactions/LeasingTransactionsSuite.scala

```diff
@@ -139,4 +140,27 @@ class LeasingTransactionsSuite extends BaseTransactionSuite with CancelAfterFail
 
     Await.result(f, waitCompletion)
   }
+
+  test("canceled flag is set correctly for lease transactions") {
```

Comment:
You could just upgrade `lease cancellation reverts eff.b. changes` test with 1 assert